### DISCUSSION TITLE
how to add a external podcast source

### DISCUSSION THREAD

**Discussion by wmoe21** [gerbera/gerbera#2879] [Q&A]
I am using the excellent Gerbera Media Server to share external Audio Live Stream Urls via UPNP inside my network.

root@raspi-pi3:/home/pi# gerbera --version
Gerbera UPnP Server version 1.1.0 - http://gerbera.io/
is running on a Raspberry Pi 3.

But how does it work to share external Podcast Folders via UPNP?

here are some Sources:

http://www.zdf.de/rss/podcast/audio/zdf/kinder/logo
https://del-podcast.podigee.io/feed/mp3
https://www.ndr.de/nachrichten/info/podcast4096.xml

I tried the latest v1.12.1-52-g12063963 but it seems thies does not work?

**Comment by KarlStraussberger** [comment]
That's a bit more complicated than it looks like. These folders are no real folders to scan. They return html that has to be parsed for content. Not undoable as we already do for appletrailers, but still a bit of interesting work.
- Download the page
- Parse it with some js code
